tools/amd_build/pyHIPIFY/hipify-python.py

```diff
@@ -437,11 +437,8 @@ def find_kernel_bounds(string):
 
         # Extract cuda kernel
         cuda_kernel = string[params[0]["start"]:parenthesis + 1]
-        cuda_kernel_dim3 = cuda_kernel
-
-        if hipify_caffe2:
-            kernel_string = string[kernel['start']:kernel['end']]
-            cuda_kernel_dim3 = add_dim3(kernel_string, cuda_kernel_dim3)
+        kernel_string = string[kernel['start']:kernel['end']]
+        cuda_kernel_dim3 = add_dim3(kernel_string, cuda_kernel)
```

Comment:
It should be `cuda_kernel_dim3` in line 445 
https://github.com/petrex/pytorch/blob/4c80a8829fe56c57cba074ad0637d7d46f132459/tools/amd_build/pyHIPIFY/hipify-python.py#L445

Reply:
@rohithkrn thanks